examples/gno.land/r/x/nir1218-evaluation-proposal/contribution.gno

```diff
@@ -0,0 +1,54 @@
+package evaluation
+
+import "std"
+
+var contributionStatus = map[string]string{}
```

Comment:
Actually, I don't understand the usage of `contributionStatus`, it really looks like a set of constants, so why using a map?

Reply:
> We currently have a bug when `map` is used in the store (see #311), the recommandation is to use `avl.Tree` instead.

Sure, I will try

Reply:
> Actually, I don't understand the usage of `contributionStatus`, it really looks like a set of constants, so why using a map?

Using a map for a faster lookup rather than iterating match if the status is valid, if I use the avl.Tree then it would be fine too.

Reply:
> Using a map for a faster lookup rather than iterating match if the status is valid, if I use the avl.Tree then it would be fine too.

Sorry, maybe I missed something, but if your keys are equals to your values, why using a map ? Also why storing it in the store ?

Reply:
> > Using a map for a faster lookup rather than iterating match if the status is valid, if I use the avl.Tree then it would be fine too.
> 
> Sorry, maybe I missed something, but if your keys are equals to your values, why using a map ? Also why storing it in the store ?

The values in the map are for reference only to validate that the status update is from a given set of statuses. I could use a map with no values as I did in pull_request.gno. Sorry, maybe I don't explain my self well.